--- a/drivers/clk/mediatek/clk-mt8163-apmixedsys.c
+++ b/drivers/clk/mediatek/clk-mt8163-apmixedsys.c
@@ -145,7 +145,7 @@ static int clk_mt8163_apmixed_probe(struct platform_device *pdev)
 	if (!clk_data)
 		return -ENOMEM;
 
-	ret = mtk_clk_register_plls(node, plls, ARRAY_SIZE(plls), clk_data);
+	ret = mtk_clk_register_plls(dev, plls, ARRAY_SIZE(plls), clk_data);
 	if (ret)
 		goto free_clk_data;
 
